grafanalib/core.py

```diff
@@ -1743,3 +1743,116 @@ def to_json_data(self):
             'transparent': self.transparent,
             'type': TABLE_TYPE,
         }
+
+
+@attr.s
+class Annotation(object):
+    """
+    Annotation create a new dashboard annotation.  Annotations can be defined to query a
+    datasource and overlay information related to key events, such as container exits or deploys.
+
+        :param default: the default value for the variable
+        :param dataSource: where to fetch the values for the variable from
```

Comment:
nit: change `dataSource` to `datasource`. It's inconsistent across grafanalib, but the grafana JSON has it as `datasource`, and I'd like to eventually make it match.